src/components/Card.js

```diff
@@ -115,8 +115,8 @@ export const Card = () => {
             </label>
           </div>
         </div>
-
-        <p>
+        <hr class="hr" />
```

Comment:
No need to add the class here, the styles you added will be applied based on element type.